### ISSUE TITLE
508-defect-3 [COGNITION]: Intercept Form - Thank you message should be one single P tag so VoiceOver reads it as one message

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#26575]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Team PM will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a VoiceOver user, I want to hear the Intercept form's confirmation message read as one single sentence, instead of three chunks that I have to advance manually.

## Details

The thank you message has several `<span>` tags nested in the paragraph, which causes VoiceOver to "chunk" them into three items. This is less clear than it could be and can be remediated by setting an appropriate width and removing the child spans.

## Acceptance Criteria

- [ ] VoiceOver reads the confirmation paragraph as a single sentence.

## Screenshots or Trace Logs

![Screen Shot 2021-06-24 at 11 43 16 AM](https://user-images.githubusercontent.com/934879/123302361-43cae580-d4e2-11eb-8238-f9ae454740b0.png)
